src/documentation/pages/components/tabs/examples/_bordered-subtabs-lumen-centurylink.vue

```diff
@@ -0,0 +1,99 @@
+<template lang="pug">
+  <ComponentTabExample title="Bordered" id="bordered-portal" additionalClasses="-bg--grey-20" padding="-p--0" :tabs="exampleTabs" :menuTabs="menuTabs" :menuId="menuId" @toggleMenuId="toggleMenuId">
+    .-p--3(:slot="$data.menuTabs[0].id")
+      .-bg--white.-p--3
+        ul.chi-tabs.-border
+          li.-active
+            a(href='#') Active tab
+          li
+            a(href='#') Tab link
+          li
+            a(href='#') Tab link
+
+    .-p--3.-bg--grey-20(:slot="$data.menuTabs[1].id")
+      .-bg--black.-p--3
```

Comment:
.-bg--white | .-bg--black must be switched conditionally as well as the class .-inverse.
Whenever possible we need to leverage Vue functionality to get rid of this kind of HTML markup manipulation.
The HTML must be rendered dynamically based on Vue component logic.
Please, apply the same to all other cases of direct HTML manipulation.